platform/qemu.go

```diff
@@ -403,6 +406,35 @@ func baseQemuArgs(board string) []string {
 	}
 }
 
+func (builder *QemuBuilder) setupUefi(secureBoot bool) error {
+	varsVariant := ""
+	if secureBoot {
+		varsVariant = ".secboot"
+	}
+	varsSrc, err := os.Open(fmt.Sprintf("/usr/share/edk2/ovmf/OVMF_VARS%s.fd", varsVariant))
```

Comment:
I'm not too familiar with this, is there a potential issue with having multiple spawned machines all touching this file?

Reply:
We're making a private writable copy; this line is just opening it read-onyl.